src/abstract/CSBondLock.sol

```diff
@@ -120,11 +122,13 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
         if (blocked < amount) {
             revert InvalidBondLockAmount();
         }
-        _changeBondLock(
-            nodeOperatorId,
-            $.bondLock[nodeOperatorId].amount - amount,
-            $.bondLock[nodeOperatorId].retentionUntil
-        );
+        unchecked {
```

Comment:
// NOTE: It's checked via `getActualLockedBond` call.

Reply:
Done